### ISSUE TITLE
Any suggestions on reproduce?

### ISSUE THREAD

**Issue by jlian2** [clovaai/voxceleb_trainer#33]
Thanks for your extensive experiment! Maybe I guess the success to a low eer is "training for 500 epoches". I believe it is diffcult for most people to reproduce that due to limit of computing resources.
Do you have any suggestions on those people? I think that should be extremely precious!

**Comment by lbjcom** [comment]
You can try [torch.manual_seed()](https://pytorch.org/docs/stable/torch.html?highlight=manual_seed#torch.manual_seed) to fix the random seed. If you add the function in the beginning you can get consistent result.

**Comment by joonson** [comment]
The training to 500 epochs does help, but the EER at 200 epochs is usually only 0.1-0.2% higher than at 500 epochs. Adjusting learning rates and other tricks such as weight averaging may help performance too.

**Comment by joonson** [comment]
Also note that the baseline models (`ResNetSE34`,`ResNetSE34L`) have only 1.4 mil params compared to 22 mil of the original ResNet-34. If you change the number of channels to match the original ResNet, it will converge much faster (e.g. in 100 epochs). The network with so few channels take more epochs to converge, but is still faster to converge in terms of time since each epoch takes much less time.
